tox.ini

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
I ran a flake8 command and it found a few things to correct (after I installed it, I ran it with no params btw):

So first off you can skip the "E501 line too long", I just don't care all that much about these.
There's a few models being imported in parse_ccv but not being used, aka "parse_ccv.py:8:1: F401 'ccv.models.ConferencePublication' imported but unused" - can you look into that?